kalite/cli.py

```diff
@@ -511,6 +509,9 @@ def start(debug=False, daemonize=True, args=[], skip_job_scheduler=False, port=N
 
     manage('initialize_kalite')
 
+    for addr in get_urls_proxy(output_pipe=sys.stdout):
+        sys.stdout.write("\t{}\n".format(addr))
+
```

Comment:
It seems you've moved the stuff that outputs to `sys.stdout`, yet kept the `print()` stuff above. Any reason for that?

I don't think there's a good reason why one of them is `sys.stdout`, while others are `print()`, but I can investigate this later...